bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/QueryResultsFrame.scala

```diff
@@ -55,12 +55,17 @@ object QueryResultsFrame extends Frame with PreferredSizeFrame {
     override val flipAction = "Plot"
   }
 
+  private object GuidingFeedbackRenderer {
+    val icons = AgsGuideQuality.All.map(q => q -> GuidingIcon(q, enabled = true)).toMap
```

Comment:
Just FYI you can use `fproduct` here, which takes `F[A] => (A => B) => F[(A, B)]` ... so

```scala
val icons = AgsGuideQuality.All.fproduct(GuidingIcon(_, enabled = true)).toMap
```